doc/sphinx/source/projects/ad7191_iio/ad7191_iio.rst

```diff
@@ -0,0 +1,102 @@
+AD7191 IIO Application
+""""""""""""""""""""""
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD7191 <https://www.analog.com/en/products/ad7191.html>`_
+
+**Supported Evaluation Boards:**
+
+* `EVAL-AD7191 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7191asdz.html>`_
+
+**Supported Carrier Boards:**
+
+* `SDP-K1  <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/sdp-k1.html#eb-overview>`_
+
+============
+Introduction
+============
+
+This page gives an overview of using the ARM platforms supported 
+firmware example with Analog Devices AD7191 Evaluation board and SDP-K1 controller board. 
+This example code leverages the ADI developed IIO (Industrial Input Output) ecosystem to 
+evaluate the AD7191 device by providing a device debug and data capture support.
+
+   .. image:: /source/tinyiiod/app_interface.png
+      :width: 600
+
+The interface used for communicating with PC based IIO clients is either Virtual Serial Or UART. 
+IIO Firmware leverages the ADI created no-os and platform driver software layers
+to communicates with IIO device.
+
+.. note::
+
+   This code has been developed and tested on the SDP-K1 Controller Board with
+   Arduino headers. However, the same code can be used with minimal modifications
+   on any STM32 enabled board which has Arduino Header support on it. 
```

Comment:
nit: We probably need to rephrase this or remove this altogether. `STM32 enabled boards` doesn't make sense!

Reply:
@jansunil @D-Disha

Reply:
Doesn't this still hold valid @SaikiranGudla ?

Reply:
It still holds valid, but Mbed enabled boards emphasizes the mbed-os uniform apis, ecosystem of boards and easy portability however, the code changes in case of STM32 will not be as minimal as mbed.

Reply:
We have to remove `enabled` word for sure, though!

Reply:
Can we say on any STM32 target instead